src/tests/ftest/recovery/ddb.py

```diff
@@ -34,11 +34,11 @@ def get_vos_file_path(self):
         If there are multiple VOS files, returns the first file obtained by "ls".
 
         Returns:
-            str: VOS file path such as /mnt/daos/<pool_uuid>/vos-0
+            str: VOS file path such as /mnt/daos0/<pool_uuid>/vos-0
 
         """
         hosts = NodeSet(self.hostlist_servers[0])
-        vos_path = os.path.join("/mnt/daos", self.pool.uuid.lower())
+        vos_path = os.path.join("/mnt/daos0", self.pool.uuid.lower())
```

Comment:
`/mnt/daos0` should not be hardcoded.  We can determine the scm_mount directory from the `DaosServerManager` class - `scm_mount = self.server_managers[0].get_config_value("scm_mount")`

Reply:
Done. Thanks.